### ISSUE TITLE
test2-suite-pm 0.000097-1 fails tests

### ISSUE THREAD

**Issue by danielj7** [fink/fink-distributions#443]
test2-suite-pm 0.000097-1 fails tests with `t/modules/Tools/Tester.t ........................ Can't locate object method "facet_key" via package "Test2::EventFacet::Info::Table" at /sw/src/fink.build/test2-suite-pm5184-0.000097-1/Test2-Suite-0.000097/blib/lib/Test2/Tools/Tester.pm line 24.`. Maybe a missing dep? Build log follows.

```
ARCHFLAGS="-I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE" /usr/bin/arch -x86_64 perl5.18 Makefile.PL PERL="/usr/bin/arch -x86_64 perl5.18" PREFIX=/sw INSTALLPRIVLIB=/sw/lib/perl5/5.18.4 INSTALLARCHLIB=/sw/lib/perl5/5.18.4/darwin-thread-multi-2level INSTALLSITELIB=/sw/lib/perl5/5.18.4 INSTALLSITEARCH=/sw/lib/perl5/5.18.4/darwin-thread-multi-2level INSTALLMAN1DIR=/sw/share/man/man1 INSTALLMAN3DIR=/sw/share/man/man3 INSTALLSITEMAN1DIR=/sw/share/man/man1 INSTALLSITEMAN3DIR=/sw/share/man/man3 INSTALLBIN=/sw/bin INSTALLSITEBIN=/sw/bin INSTALLSCRIPT=/sw/bin
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Test2::Suite
Writing MYMETA.yml and MYMETA.json
/usr/bin/make CC=gcc CXX=g++
cp lib/Test2/Bundle.pm blib/lib/Test2/Bundle.pm
cp lib/Test2/Workflow/Build.pm blib/lib/Test2/Workflow/Build.pm
cp lib/Test2/Compare/Wildcard.pm blib/lib/Test2/Compare/Wildcard.pm
cp lib/Test2/Compare/Hash.pm blib/lib/Test2/Compare/Hash.pm
cp lib/Test2/Util/Table/Cell.pm blib/lib/Test2/Util/Table/Cell.pm
cp lib/Test2/Util/Times.pm blib/lib/Test2/Util/Times.pm
cp lib/Test2/Tools/GenTemp.pm blib/lib/Test2/Tools/GenTemp.pm
cp lib/Test2/Util/Table.pm blib/lib/Test2/Util/Table.pm
cp lib/Test2/Plugin/SRand.pm blib/lib/Test2/Plugin/SRand.pm
cp lib/Test2/Todo.pm blib/lib/Test2/Todo.pm
cp lib/Test2/Util/Ref.pm blib/lib/Test2/Util/Ref.pm
cp lib/Test2/Workflow/BlockBase.pm blib/lib/Test2/Workflow/BlockBase.pm
cp lib/Test2/Manual/Testing/Migrating.pm blib/lib/Test2/Manual/Testing/Migrating.pm
cp lib/Test2/Compare/Undef.pm blib/lib/Test2/Compare/Undef.pm
cp lib/Test2/AsyncSubtest.pm blib/lib/Test2/AsyncSubtest.pm
cp lib/Test2/Compare/Set.pm blib/lib/Test2/Compare/Set.pm
cp lib/Test2/Compare/Regex.pm blib/lib/Test2/Compare/Regex.pm
cp lib/Test2/Tools/Exception.pm blib/lib/Test2/Tools/Exception.pm
cp lib/Test2/Bundle/More.pm blib/lib/Test2/Bundle/More.pm
cp lib/Test2/Workflow/Runner.pm blib/lib/Test2/Workflow/Runner.pm
cp lib/Test2/Compare/Number.pm blib/lib/Test2/Compare/Number.pm
cp lib/Test2/Tools/Tester.pm blib/lib/Test2/Tools/Tester.pm
cp lib/Test2/Tools/Mock.pm blib/lib/Test2/Tools/Mock.pm
cp lib/Test2/AsyncSubtest/Formatter.pm blib/lib/Test2/AsyncSubtest/Formatter.pm
cp lib/Test2/Manual/Testing/Todo.pm blib/lib/Test2/Manual/Testing/Todo.pm
cp lib/Test2/Util/Sub.pm blib/lib/Test2/Util/Sub.pm
cp lib/Test2/Compare/EventMeta.pm blib/lib/Test2/Compare/EventMeta.pm
cp lib/Test2/Tools/Class.pm blib/lib/Test2/Tools/Class.pm
cp lib/Test2/AsyncSubtest/Event/Detach.pm blib/lib/Test2/AsyncSubtest/Event/Detach.pm
cp lib/Test2/Tools/Exports.pm blib/lib/Test2/Tools/Exports.pm
cp lib/Test2/Util/Stash.pm blib/lib/Test2/Util/Stash.pm
cp lib/Test2/Tools/Compare.pm blib/lib/Test2/Tools/Compare.pm
cp lib/Test2/Tools/AsyncSubtest.pm blib/lib/Test2/Tools/AsyncSubtest.pm
cp lib/Test2/Manual/Tooling/Nesting.pm blib/lib/Test2/Manual/Tooling/Nesting.pm
cp lib/Test2/Manual/Anatomy/Event.pm blib/lib/Test2/Manual/Anatomy/Event.pm
cp lib/Test2/Manual/Tooling/Testing.pm blib/lib/Test2/Manual/Tooling/Testing.pm
cp lib/Test2/Bundle/Simple.pm blib/lib/Test2/Bundle/Simple.pm
cp lib/Test2/Compare/Base.pm blib/lib/Test2/Compare/Base.pm
cp lib/Test2/Tools/Event.pm blib/lib/Test2/Tools/Event.pm
cp lib/Test2/AsyncSubtest/Hub.pm blib/lib/Test2/AsyncSubtest/Hub.pm
cp lib/Test2/Require/Perl.pm blib/lib/Test2/Require/Perl.pm
cp lib/Test2/Manual/Anatomy/EndToEnd.pm blib/lib/Test2/Manual/Anatomy/EndToEnd.pm
cp lib/Test2/Plugin.pm blib/lib/Test2/Plugin.pm
cp lib/Test2/Manual/Anatomy.pm blib/lib/Test2/Manual/Anatomy.pm
cp lib/Test2/Require/Fork.pm blib/lib/Test2/Require/Fork.pm
cp lib/Test2/V0.pm blib/lib/Test2/V0.pm
cp lib/Test2/Tools/Warnings.pm blib/lib/Test2/Tools/Warnings.pm
cp lib/Test2/Tools/Spec.pm blib/lib/Test2/Tools/Spec.pm
cp lib/Test2/Plugin/Times.pm blib/lib/Test2/Plugin/Times.pm
cp lib/Test2/Compare/Object.pm blib/lib/Test2/Compare/Object.pm
cp lib/Test2/Tools/Ref.pm blib/lib/Test2/Tools/Ref.pm
cp lib/Test2/Suite.pm blib/lib/Test2/Suite.pm
cp lib/Test2/Plugin/ExitSummary.pm blib/lib/Test2/Plugin/ExitSummary.pm
cp lib/Test2/Require/AuthorTesting.pm blib/lib/Test2/Require/AuthorTesting.pm
cp lib/Test2/Manual/Testing/Introduction.pm blib/lib/Test2/Manual/Testing/Introduction.pm
cp lib/Test2/AsyncSubtest/Event/Attach.pm blib/lib/Test2/AsyncSubtest/Event/Attach.pm
cp lib/Test2/Workflow/Task.pm blib/lib/Test2/Workflow/Task.pm
cp lib/Test2/Manual/Testing/Planning.pm blib/lib/Test2/Manual/Testing/Planning.pm
cp lib/Test2/Compare/Scalar.pm blib/lib/Test2/Compare/Scalar.pm
cp lib/Test2/Workflow.pm blib/lib/Test2/Workflow.pm
cp lib/Test2/Tools/Target.pm blib/lib/Test2/Tools/Target.pm
cp lib/Test2/Compare/String.pm blib/lib/Test2/Compare/String.pm
cp lib/Test2/Require/Module.pm blib/lib/Test2/Require/Module.pm
cp lib/Test2/Bundle/Extended.pm blib/lib/Test2/Bundle/Extended.pm
cp lib/Test2/Compare/Pattern.pm blib/lib/Test2/Compare/Pattern.pm
cp lib/Test2/Workflow/Task/Action.pm blib/lib/Test2/Workflow/Task/Action.pm
cp lib/Test2/Manual/Testing.pm blib/lib/Test2/Manual/Testing.pm
cp lib/Test2/Tools/Basic.pm blib/lib/Test2/Tools/Basic.pm
cp lib/Test2/Manual/Tooling.pm blib/lib/Test2/Manual/Tooling.pm
cp lib/Test2/Manual/Concurrency.pm blib/lib/Test2/Manual/Concurrency.pm
cp lib/Test2/Manual.pm blib/lib/Test2/Manual.pm
cp lib/Test2/Compare/Delta.pm blib/lib/Test2/Compare/Delta.pm
cp lib/Test2/Util/Table/LineBreak.pm blib/lib/Test2/Util/Table/LineBreak.pm
cp lib/Test2/Plugin/DieOnFail.pm blib/lib/Test2/Plugin/DieOnFail.pm
cp lib/Test2/Compare/Bag.pm blib/lib/Test2/Compare/Bag.pm
cp lib/Test2/Require/EnvVar.pm blib/lib/Test2/Require/EnvVar.pm
cp lib/Test2/Compare/Ref.pm blib/lib/Test2/Compare/Ref.pm
cp lib/Test2/Util/Grabber.pm blib/lib/Test2/Util/Grabber.pm
cp lib/Test2/Plugin/BailOnFail.pm blib/lib/Test2/Plugin/BailOnFail.pm
cp lib/Test2/Manual/Tooling/FirstTool.pm blib/lib/Test2/Manual/Tooling/FirstTool.pm
cp lib/Test2/Compare/Custom.pm blib/lib/Test2/Compare/Custom.pm
cp lib/Test2/Require/RealFork.pm blib/lib/Test2/Require/RealFork.pm
cp lib/Test2/Compare/Negatable.pm blib/lib/Test2/Compare/Negatable.pm
cp lib/Test2/Compare/Bool.pm blib/lib/Test2/Compare/Bool.pm
cp lib/Test2/Mock.pm blib/lib/Test2/Mock.pm
cp lib/Test2/Tools.pm blib/lib/Test2/Tools.pm
cp lib/Test2/Compare/Float.pm blib/lib/Test2/Compare/Float.pm
cp lib/Test2/Workflow/Task/Group.pm blib/lib/Test2/Workflow/Task/Group.pm
cp lib/Test2/Compare/Event.pm blib/lib/Test2/Compare/Event.pm
cp lib/Test2/Compare/OrderedSubset.pm blib/lib/Test2/Compare/OrderedSubset.pm
cp lib/Test2/Require/Threads.pm blib/lib/Test2/Require/Threads.pm
cp lib/Test2/Compare/DeepRef.pm blib/lib/Test2/Compare/DeepRef.pm
cp lib/Test2/Tools/Subtest.pm blib/lib/Test2/Tools/Subtest.pm
cp lib/Test2/Plugin/UTF8.pm blib/lib/Test2/Plugin/UTF8.pm
cp lib/Test2/Compare/Meta.pm blib/lib/Test2/Compare/Meta.pm
cp lib/Test2/Manual/Contributing.pm blib/lib/Test2/Manual/Contributing.pm
cp lib/Test2/Compare.pm blib/lib/Test2/Compare.pm
cp lib/Test2/Event/Times.pm blib/lib/Test2/Event/Times.pm
cp lib/Test2/Tools/Encoding.pm blib/lib/Test2/Tools/Encoding.pm
cp lib/Test2/Tools/Grab.pm blib/lib/Test2/Tools/Grab.pm
cp lib/Test2/Compare/Array.pm blib/lib/Test2/Compare/Array.pm
cp lib/Test2/Require.pm blib/lib/Test2/Require.pm
cp lib/Test2/Manual/Tooling/Subtest.pm blib/lib/Test2/Manual/Tooling/Subtest.pm
cp lib/Test2/Util/Term.pm blib/lib/Test2/Util/Term.pm
cp lib/Test2/Tools/ClassicCompare.pm blib/lib/Test2/Tools/ClassicCompare.pm
cp lib/Test2/Tools/Defer.pm blib/lib/Test2/Tools/Defer.pm
Manifying 102 pod documents
sudo -u fink-bld [ENV] sh -c /tmp/fink.H6SW4
/usr/bin/make test || exit 2
PERL_DL_NONLAZY=1 "/usr/bin/arch" -x86_64 perl5.18 "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t t/acceptance/*.t t/behavior/*.t t/modules/*.t t/modules/AsyncSubtest/*.t t/modules/AsyncSubtest/Event/*.t t/modules/Bundle/*.t t/modules/Compare/*.t t/modules/Plugin/*.t t/modules/Require/*.t t/modules/Tools/*.t t/modules/Util/*.t t/modules/Util/Table/*.t t/modules/Workflow/*.t t/modules/Workflow/Task/*.t t/regression/*.t
t/00-report.t ................................... # 
# DIAGNOSTICS INFO IN CASE OF FAILURE:
# +------+----------+
# | perl | 5.018004 |
# +------+----------+
# +-----------------+-----------+
# | CAPABILITY      | SUPPORTED |
# +-----------------+-----------+
# | CAN_FORK        | Yes       |
# | CAN_REALLY_FORK | Yes       |
# | CAN_THREAD      | Yes       |
# +-----------------+-----------+
# +-------------------+----------+
# | DEPENDENCY        | VERSION  |
# +-------------------+----------+
# | B                 | 1.42_02  |
# | Carp              | 1.29     |
# | Exporter          | 5.70     |
# | Importer          | 0.024    |
# | Module::Pluggable | 5.2      |
# | Scalar::Util      | 1.5      |
# | Scope::Guard      | 0.21     |
# | Sub::Info         | 0.002    |
# | Term::Table       | 0.012    |
# | Test2             | 1.302164 |
# | Time::HiRes       | 1.9725   |
# | overload          | 1.22     |
# | utf8              | 1.10     |
# +-------------------+----------+
# +--------------------+---------+
# | OPTIONAL           | VERSION |
# +--------------------+---------+
# | Sub::Name          | 0.21    |
# | Term::ReadKey      | 2.37    |
# | Term::Size::Any    | N/A     |
# | Unicode::GCString  | N/A     |
# | Unicode::LineBreak | N/A     |
# +--------------------+---------+
t/00-report.t ................................... ok   
t/acceptance/OO.t ............................... ok    
t/acceptance/skip.t ............................. ok   
t/acceptance/Tools.t ............................ ok   
t/acceptance/Workflow-Acceptance.t .............. ok    
t/acceptance/Workflow-Acceptance2.t ............. ok   
t/acceptance/Workflow-Acceptance3.t ............. ok   
t/acceptance/Workflow-Acceptance4.t ............. ok   
t/acceptance/Workflow-Acceptance5.t ............. ok   
t/behavior/filtering.t .......................... ok   
t/behavior/Mocking.t ............................ ok   
t/behavior/no_done_testing.t .................... ok   
t/behavior/no_leaks_any.t ....................... ok   
t/behavior/no_leaks_no_fork.t ................... ok   
t/behavior/no_leaks_no_iso.t .................... ok   
t/behavior/no_leaks_no_threads.t ................ ok   
t/behavior/simple.t ............................. ok   
t/load_manual.t ................................. ok   
t/modules/AsyncSubtest.t ........................ ok    
t/modules/AsyncSubtest/Event/Attach.t ........... ok   
t/modules/AsyncSubtest/Event/Detach.t ........... ok   
t/modules/AsyncSubtest/Hub.t .................... ok   
t/modules/Bundle.t .............................. ok   
t/modules/Bundle/Extended.t ..................... ok   
t/modules/Bundle/More.t ......................... ok   
t/modules/Bundle/Simple.t ....................... ok   
t/modules/Compare.t ............................. ok    
t/modules/Compare/Array.t ....................... ok   
t/modules/Compare/Bag.t ......................... ok   
t/modules/Compare/Base.t ........................ ok    
t/modules/Compare/Bool.t ........................ ok   
t/modules/Compare/Custom.t ...................... ok    
t/modules/Compare/Delta.t ....................... ok    
t/modules/Compare/Event.t ....................... ok   
t/modules/Compare/EventMeta.t ................... ok   
t/modules/Compare/Float.t ....................... ok    
t/modules/Compare/Hash.t ........................ ok   
t/modules/Compare/Meta.t ........................ ok   
t/modules/Compare/Number.t ...................... ok   
t/modules/Compare/Object.t ...................... ok   
t/modules/Compare/OrderedSubset.t ............... ok   
t/modules/Compare/Pattern.t ..................... ok    
t/modules/Compare/Ref.t ......................... ok    
t/modules/Compare/Regex.t ....................... ok    
t/modules/Compare/Scalar.t ...................... ok    
t/modules/Compare/Set.t ......................... ok   
t/modules/Compare/String.t ...................... ok   
t/modules/Compare/Undef.t ....................... ok   
t/modules/Compare/Wildcard.t .................... ok   
t/modules/Mock.t ................................ ok    
t/modules/Plugin.t .............................. ok   
t/modules/Plugin/BailOnFail.t ................... ok   
t/modules/Plugin/DieOnFail.t .................... ok   
t/modules/Plugin/ExitSummary.t .................. ok   
t/modules/Plugin/SRand.t ........................ ok    
t/modules/Plugin/Times.t ........................ ok   
t/modules/Plugin/UTF8.t ......................... ok   
t/modules/Require.t ............................. ok   
t/modules/Require/AuthorTesting.t ............... ok   
t/modules/Require/EnvVar.t ...................... ok   
t/modules/Require/Fork.t ........................ ok   
t/modules/Require/Module.t ...................... ok   
t/modules/Require/Perl.t ........................ ok   
t/modules/Require/RealFork.t .................... ok   
t/modules/Require/Threads.t ..................... ok   
t/modules/Suite.t ............................... ok   
t/modules/Todo.t ................................ ok    
t/modules/Tools.t ............................... ok   
t/modules/Tools/AsyncSubtest.t .................. ok   
t/modules/Tools/Basic.t ......................... ok    
t/modules/Tools/Class.t ......................... ok   
t/modules/Tools/ClassicCompare.t ................ ok    
t/modules/Tools/ClassicCompare2.t ............... ok   
t/modules/Tools/Compare.t ....................... ok    
t/modules/Tools/Defer.t ......................... ok    
t/modules/Tools/Encoding.t ...................... ok   
t/modules/Tools/Event.t ......................... ok   
t/modules/Tools/Exception.t ..................... ok    
t/modules/Tools/Exports.t ....................... ok   
t/modules/Tools/GenTemp.t ....................... ok    
t/modules/Tools/Grab.t .......................... ok   
t/modules/Tools/Mock.t .......................... ok    
t/modules/Tools/Ref.t ........................... ok   
t/modules/Tools/Spec.t .......................... skipped: Tests not yet written
t/modules/Tools/Subtest.t ....................... ok    
t/modules/Tools/Target.t ........................ ok   
t/modules/Tools/Tester.t ........................ Can't locate object method "facet_key" via package "Test2::EventFacet::Info::Table" at /sw/src/fink.build/test2-suite-pm5184-0.000097-1/Test2-Suite-0.000097/blib/lib/Test2/Tools/Tester.pm line 24.
Compilation failed in require at t/modules/Tools/Tester.t line 2.
BEGIN failed--compilation aborted at t/modules/Tools/Tester.t line 2.
t/modules/Tools/Tester.t ........................ Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
t/modules/Tools/Warnings.t ...................... ok    
t/modules/Util/Grabber.t ........................ ok   
t/modules/Util/Ref.t ............................ ok    
t/modules/Util/Stash.t .......................... ok    
t/modules/Util/Sub.t ............................ ok   
t/modules/Util/Table.t .......................... ok   
t/modules/Util/Table/Cell.t ..................... ok   
t/modules/Util/Table/LineBreak.t ................ ok   
t/modules/Util/Times.t .......................... ok   
t/modules/V0.t .................................. ok   
t/modules/Workflow.t ............................ skipped: Tests not yet written
t/modules/Workflow/BlockBase.t .................. skipped: Tests not yet written
t/modules/Workflow/Build.t ...................... skipped: Tests not yet written
t/modules/Workflow/Runner.t ..................... skipped: Tests not yet written
t/modules/Workflow/Task.t ....................... skipped: Tests not yet written
t/modules/Workflow/Task/Action.t ................ ok   
t/modules/Workflow/Task/Group.t ................. skipped: Tests not yet written
t/regression/10-set_and_dne.t ................... ok   
t/regression/132-bool.t ......................... skipped: Author test, set the $AUTHOR_TESTING environment variable to run it
t/regression/27-1-Test2-Bundle-More.t ........... ok   
t/regression/27-2-Test2-Tools-Compare.t ......... ok   
t/regression/27-3-Test2-Tools-ClassicCompare.t .. ok   
t/regression/43-bag-on-empty.t .................. ok   
t/regression/Test2-Mock.t ....................... ok   
t/regression/Test2-Tools-Class.t ................ ok   
t/regression/todo_and_facets.t .................. ok    
t/regression/utf8-mock.t ........................ ok   

Test Summary Report
-------------------
t/modules/Tools/Tester.t                      (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=114, Tests=731, 11 wallclock secs ( 0.28 usr  0.15 sys +  9.93 cusr  2.07 csys = 12.43 CPU)
Result: FAIL
Failed 1/114 test programs. 0/731 subtests failed.
make: *** [test_dynamic] Error 255
```

**Comment by nieder** [comment]
Upstream bug report Test-More/Test2-Suite#175 which fixes this failure in newer releases.

**Comment by dmacks** [comment]
Fixed by 9a5f6ce (new test2-suite-pm version with upgraded dependencies)
